.trybe/requirements.json

```
@@ -1,59 +1,59 @@
+      "description": "2 - Adicione abaixo do título um pequeno e discreto parágrafo com id=\"funcionamento\" e com o texto \"Clique duas vezes em um item para marcá-lo como completo\"",
       "bonus": false
     },
     {
-      "description": "Deve haver um input com o id=\"texto-tarefa\" onde o usuário poderá digitar o nome do item que deseja adicionar à lista",
+      "description": "3 - Adicione um input com o id=\"texto-tarefa\" onde o usuário poderá digitar o nome do item que deseja adicionar à lista",
       "bonus": false
     },
     {
-      "description": "Deve haver uma lista ordenada de tarefas com o id=\"lista-tarefas\"",
+      "description": "4 - Adicione uma lista ordenada de tarefas com o id=\"lista-tarefas\"",
       "bonus": false
     },
     {
-      "description": "Deve haver um botão com id=\"criar-tarefa\" e, ao clicar nesse botão, um novo item deverá ser criado ao final da lista e o texto do input deve ser limpo",
+      "description": "5 - Adicione um botão com id=\"criar-tarefa\" e, ao clicar nesse botão, um novo item deverá ser criado ao final da lista e o texto do input deve ser limpo",
       "bonus": false
     },
     {
-      "description": "Os itens da lista de tarefas devem ser ordenados por ordem de criação",
+      "description": "6 - Ordene os itens da lista de tarefas eles devem ser ordenados por ordem de criação",
       "bonus": false
     },
     {
-      "description": "Ao clicar em um item da lista, altere a cor de fundo do item para cinza rgb(128,128,128)",
+      "description": "7 - Clicar em um item da lista deve alterar a cor de fundo do item para cinza rgb(128,128,128)",
       "bonus": false
     },
     {
-      "description": "Não deve ser possível selecionar mais de um elemento da lista ao mesmo tempo",
+      "description": "8 - Não deve ser possível selecionar mais de um elemento da lista ao mesmo tempo",
       "bonus": false
     },
     {
-      "description": "Ao clicar duas vezes em um item, ele deverá ser riscado, indicando que foi completo. Deve ser possível desfazer essa ação clicando novamente duas vezes no item",
+      "description": "9 - Clicar duas vezes em um item, faz com que ele seja riscado, indicando que foi completo. Deve ser possível desfazer essa ação clicando novamente duas vezes no item",
       "bonus": false
     },
     {
-      "description": "Deve haver um botão com `id=\"apaga-tudo\"`que quando clicado deve apagar todos os itens da lista",
+      "description": "10 - Adicione um botão com id=\"apaga-tudo\" que quando clicado deve apagar todos os itens da lista",
       "bonus": false
     },
     {
-      "description": "Deve haver um botão`id=\"remover-finalizados\"`que quando clicado remove **somente** os elementos finalizados da sua lista",
+      "description": "11 - Adicione um botão id=\"remover-finalizados\" que quando clicado remove **somente** os elementos finalizados da sua lista",
```

O que acha de deixar no mesmo padrão do requisito anterior e dos outros abaixo?
```suggestion
      "description": "11 - Adicione um botão com id=\"remover-finalizados\" que quando clicado remove **somente** os elementos finalizados da sua lista",
```